package.json

```diff
@@ -36,7 +36,8 @@
     "postspec": "npm run clean",
     "clean": "rm -rf test/fixtures/test*.js",
     "web": "node web",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+    "semantic-release": "semantic-release pre && npm publish && semantic-release post",
+    "postinstall": "node -e \"console.log('\\u001b[32mLove nodemon? You can now support the project via the open collective:\\u001b[22m\\u001b[39m\\n > \\u001b[96m\\u001b[1mhttps://opencollective.com/nodemon/donate\\u001b[0m\\n')\""
```

Comment:
I understand your needs but this is really clumsy...

Imagines if all npm modules do that. :-/

Can you deactivate this log when we are in CI env?

Reply:
If all npm modules did this, then I'd find another way to ask and plea for help.

I've found myself asking this a lot lately, but: why are you using nodemon in CI?

Reply:
Just installing packages. 

Can you considers to move this console.log when nodemon is running for the first time ?

Reply:
As an arbitrary aim, if the project can raise $10-20 per week, then I'll happily remove the plea for support (note this wouldn't cover the past costs or even my ongoing costs, but it shows there's support that values nodemon).